### PR TITLE
Add garimadhakal nepal recommendations

### LINKED ISSUE #24: add garimadhakal nepal file
- [x] Create a branch
- [x] Add file/commits
- [x] Open a PR
- [x] Have a discussion about the PR (Collaborate)
- [x] Make edits
- [x] Get :shipit: approval
- [x] Merge PR

### PR DESCRIPTION
**Pull request by ghost**
close #24 

hey @silpagadi, check this out! Does this look like a good list of recommendations? 👍 

cc: @beardofedu

### CODE REVIEW

File: garimadhakal-nepal.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+I'm from Nepal!
+
+#Kathmandu, Nepal
+
+##Places to Eat
+- Roadhouse Cafe
+
+##Places to Go
+- Durbar square
```
Comment (silpagadi) [inline garimadhakal-nepal.md]: I will make sure to go to Durbar square if i visit Nepal.

**Comment by silpagadi** [comment]
I will make sure to go to Durbar square if i visit Nepal.

**Comment by ghost** [comment]
@silpagadi Thanks. I also added a new place to go. Does this look good to :shipit: 🐐 ?

**Comment by silpagadi** [comment]
@garimadhakal 
Great! :ship:

**Pull request merged**